pkg/sfu/downtrack.go

```diff
@@ -627,19 +627,21 @@ func (d *DownTrack) keyFrameRequester() {
 	}
 
 	timer := time.NewTimer(math.MaxInt64)
+	timer.Stop()
+
 	defer timer.Stop()
 
 	for !d.IsClosed() {
-		if !timer.Stop() {
```

Comment:
if the timer fired in the select below this returns false but we've already drained the channel...